add_user.php

```diff
@@ -54,7 +54,7 @@
 									?>
 									<center>
 										<h1>ADD USER</h1>
-										<h4><form method = "post" id="add_user.php">
+										<h3><form method = "post" id="add_user.php">
```

Comment:
I removed h4 styling as I didn't need that many headings so I changed all text that was h4 to h3